scripts/feedme.coffee

```diff
@@ -104,15 +109,78 @@ module.exports = (robot) ->
 
   fabrik = new Fabrik(robot)
   ernis = new Ernis(robot)
-  now = moment()
-  # ranging from 1 (Monday) to 7 (Sunday)
-  day = now.isoWeekday()
+
+  # note that we can't store the reminders anywhere because node.js returns a
+  # Timeout object in contrast to the usual (browser) integer which is then
+  # needed to clear the timeout later on (if there is a new reminder set). This
+  # Timeout object can't be JSON.stringified (circular reference) and thus not
+  # stored. As a result reminders are lost on restart of hubot.
+  reminder = {}
+
+  # wait for given time and then send a message to #vie-food with all
+  # subscribed users for this food type.
+  setReminder = (foodType, hour = FOOD_TIME.HOUR, minute = FOOD_TIME.MINUTE) ->
+    setTimeout ( =>
+      eater = robot.brain.get "feedme.eater"
+      people = if eater and foodType of eater then eater[foodType] else []
+      msg = if people.length > 0 then "#{people.toString()}:\n" else ""
+      robot.messageRoom "vie-food", msg + "Los los ... #{foodType} wartet nicht!"
+      delete reminder[foodType]
+      # clear all eaters, they're supposed to be fed.
+      delete eater[foodType]
+      robot.brain.set "feedme.eater", eater
+    ), moment().hour(hour).minute(minute).seconds(0)
+      .diff(moment(), 'milliseconds')
+
+  existsReminder = (foodType) ->
+    foodType of reminder
+
+  # clear all reservations + reminders @midnight
```

Comment:
Not sure this is stable enough. Why not clear right after sending out the reminder?